.gitignore

```
@@ -7,3 +7,4 @@
 # Python egg metadata, regenerated from source files by setuptools.
 /*.egg-info
 .eggs
+asterism/.DS_Store
```

```suggestion
*.DS_Store
```
Does that work? That way we catch all the DS_Stores...